### PR TITLE
Release notes for ACP/AEP react native core 

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## July 25, 2022
+
+### ACP React Native Core 2.0.2
+
+* Targeting android 12 (API 31) for the Android implementation
```
Comment (calebk1) [inline release-notes/README.md:11]: Android 12 should be a capital A.

Comment (calebk1) [reply release-notes/README.md:11]: Also, missing a period at the end of the sentence.

**Review by calebk1 — APPROVED** [review]
Just a brief fix (for both of them!)

File: release-notes/README.md (hunk 1 above)
Comment (yangyansong-adbe) [reply release-notes/README.md]: The PR is updated.

**Review by calebk1 — APPROVED** [review]

**Pull request merged**